lib/foreman_inventory_upload/async/shell_process.rb

```diff
@@ -8,24 +8,30 @@ class ShellProcess < ::ApplicationJob
       def perform(instance_label)
         klass_name = self.class.name
         logger.debug("Starting #{klass_name} with label #{instance_label}")
-        progress_output = ProgressOutput.register(instance_label)
-        Open3.popen2e(hash_to_s(env), *preprocess_command(command)) do |_stdin, stdout_stderr, wait_thread|
-          progress_output.status = "Running in pid #{wait_thread.pid}"
+        progress_output_for(instance_label) do |progress_output|
+          Open3.popen2e(hash_to_s(env), *preprocess_command(command)) do |stdin, stdout_stderr, wait_thread|
```

Comment:
```suggestion
          Open3.popen2e(hash_to_s(env), *preprocess_command(command)) do |_stdin, stdout_stderr, wait_thread|
```
we do not use `stdin` parameter.

Reply:
interesting why `rubocop` didn't catch it either